include/RMGTrackOutputScheme.hh

```diff
@@ -57,12 +92,32 @@ class RMGTrackOutputScheme : public RMGVOutputScheme {
 
     bool fStoreSinglePrecisionEnergy = false;
     bool fStoreSinglePrecisionPosition = false;
+    bool fStoreAlways = false;
 
     std::map<std::string, uint32_t> fProcessMap;
+    std::set<int> fStoredProcessIDs; // This set keeps track of the process IDs that have not been discarded.
 
     std::set<std::string> fFilterProcess;
     std::set<int> fFilterParticle;
     double fFilterEnergy = -1;
+
+    struct RMGTrackEntry {
+        int eventId;
+        int trackId;
+        int parentId;
+        int procId;
+        int particlePdg;
+        double globalTime;
+        double xPosition;
+        double yPosition;
+        double zPosition;
+        double px;
+        double py;
+        double pz;
+        double kineticEnergy;
```

Comment:
We usually use snake case for remage struct members, see e.g.

https://github.com/legend-exp/remage/blob/bd59e2644987026e02af10a4f92cf82914bc5654/include/RMGVertexConfinement.hh#L50-L63